pkg/k8s/apis/cilium.io/v2alpha1/bgp_node_override_types.go

```diff
@@ -40,11 +42,6 @@ type CiliumBGPNodeConfigOverrideList struct {
 }
 
 type CiliumBGPNodeConfigOverrideSpec struct {
-	// NodeRef is the name of the node for which the BGP configuration is overridden.
```

Comment:
Is there ever a case where a given node would have more than one override?

Is there a case where we would like to apply overrides by a label selector on nodes, rather than by name?